atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -247,14 +247,15 @@ public boolean hasNext() {
                         runningSize += sizingFunction.apply(firstEntry);
                         entries.add(firstEntry);
                         if (runningSize > maximumBytesPerPartition && log.isWarnEnabled()) {
-                            String message = "Encountered an entry of approximate size {} bytes, "
+                            final String message = "Encountered an entry of approximate size {} bytes, "
```

Comment:
Do we even need the final now? I think the findbugs plugin has a different level of contraint, or my knowledge might be about an outdated version.. Please apply the plugin first.